src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,46 @@
 import React from 'react';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import { Post } from '../../types/Post';
+
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    user,
+    body,
+    title,
+    comments,
+  } = post;
+
+  return (
+    <>
+      <div className="PostInfo">
+        <div className="PostInfo__header">
+          <h3 className="PostInfo__title">{title}</h3>
+
+          <p>
+            {' Posted by  '}
+
+            {user && <UserInfo user={user} />}
+          </p>
```

Comment:
Same here. You can just add text in tag
```suggestion
          <p>Posted by {user && <UserInfo user={user} />}</p>
```

Reply:
thank you, I have to use this method due to the linter requirements
space is vanished when I wrap the lines

<img width="410" alt="img" src="https://user-images.githubusercontent.com/120301342/215835236-0cab7f7f-c23e-4f61-bcfe-7041e994d180.PNG">